src/jshint.js

```diff
@@ -3561,12 +3561,13 @@ var JSHINT = (function() {
     return classdef.call(this, true);
   });
 
-  function classdef(stmt) {
+  function classdef(isStatement) {
+
     /*jshint validthis:true */
     if (!state.option.inESNext()) {
       warning("W104", state.tokens.curr, "class");
     }
-    if (stmt) {
+    if (isStatement) {
```

Comment:
This is unrelated, but I couldn't help myself. Feel free to reject, I'll update accordingly.